test/apicoverage-webhook/coverage_rules.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"strings"
+
+	"github.com/knative/test-infra/tools/webhook-apicoverage/resourcetree"
+)
+
+// coverage_rules.go contains all the apicoverage rules specified for knative serving.
```

Comment:
These seem pretty generic rules that could be part of a shared collection of rules, which are then used by each individual repo (getNodeRules and getFieldRules can be moved to webhook_handlers.go).

Reply:
Actually they are not generic. Some repo might have their own reasons to have their own. I would prefer moving it to test-infra till we see the usage pattern of the next repo and then refactor rather than doing it now.

Reply:
I have added TODO and created this issue for re-evaluation: https://github.com/knative/test-infra/issues/448